build.gradle

```
@@ -177,6 +177,9 @@ publishing {
 }
 
 signing {
+    def signingKey = System.getenv("PGP_SECRET")
+    def signingPassword = System.getenv("PGP_PASSPHRASE")
```

shall we throw here if variables are not defined?